city_scrapers/spiders/chi_ssa_18.py

```diff
@@ -0,0 +1,117 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa18Spider(CityScrapersSpider):
+    name = "chi_ssa_18"
+    agency = "Chicago Special Service Area #18 North Halsted"
+    timezone = "America/Chicago"
+    start_urls = ["https://northalsted.com/community/"]
+    location = {
+        'name': 'Center on Halsted',
+        'address': '3656 N Halsted St, Chicago IL 60613, Conference Room 200 on the 2nd Floor'
+    }
+    meeting_year = ''  # This is used to track year in header elements & append to meetings
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        table = response.xpath('//*[@id="page_content_wrapper"]/div/div/div/div[1]/div[3]')
+
+        self._validate_location(table)
+        self._validate_meeting_times(table)
+
+        for item in table.xpath('.//h3 | .//p'):
+            if 'h3' in item.get():
+                ChiSsa18Spider.meeting_year = item.xpath('descendant-or-self::text()')\
+                    .re_first(r'\d\d\d\d')
+                continue
+            if '<p>' in item.get():
+                split_test = item.get().split("<br>")
```

Comment:
You might want to change the variable name here, and after splitting on `<br>` you should convert the strings back into selectors so that you can query them as HTML. Here's an example:

https://github.com/City-Bureau/city-scrapers-cle/blob/1b3d7d84c073249d5cedb2eb84f2b7eac7ece527/city_scrapers/spiders/cuya_soldiers_sailors_monument.py#L24-L27

Reply:
updated